korgan/prepayment_runtime.py

```
@@ -0,0 +1,332 @@
+        await pretrial_runtime._generate(message, state)
+        return
+    if kind == "pretrial_response":
+        await pretrial_response_runtime._generate(message, state)
+        return
+    if kind == "response":
+        await universal_document_runtime._send_response(message, state)
+        return
+    if kind == "contract":
+        await universal_document_runtime._send_contract(message, state)
+        return
+    raise ValueError(f"Unsupported paid document kind: {kind}")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)